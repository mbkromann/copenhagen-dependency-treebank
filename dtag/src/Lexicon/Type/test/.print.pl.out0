a: super=[] lvars=[]
a_a: super=[a] lvars=[]
b: super=[] lvars=[]
c: super=[] lvars=[]
abc: super=[a b c] lvars=[]
d: super=[abc a_a] lvars=[]
