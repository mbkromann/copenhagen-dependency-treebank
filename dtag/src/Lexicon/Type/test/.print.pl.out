ERROR! cannot read file /opt/dta/arcs.header in Graph->readfile
check that DTAGHOME is set correctly!
ERROR! cannot read file /opt/dta/arcs.trailer in Graph->readfile
check that DTAGHOME is set correctly!
a: super=[] lvars=[]
a_a: super=[a] lvars=[]
b: super=[] lvars=[]
c: super=[] lvars=[]
abc: super=[a b c] lvars=[]
d: super=[abc a_a] lvars=[]
